tox.ini

```diff
@@ -1,13 +1,38 @@
 [tox]
 envlist =
-    py27,py34,py35,py36,pypy,pypy3,docs
+    lint,
+    py27,
+    pypy,
+    py34,
+    py35,
+    py36,
+    pypy3,
 
 [testenv]
+usedevelop = true
+deps =
+    zope.testrunner
 commands =
     zope-testrunner --test-path=src []
     sphinx-build -b doctest -d {envdir}/.cache/doctrees docs {envdir}/.cache/doctest
+extras =
+    test
+    docs
+
+[testenv:lint]
+basepython = python3
+skip_install = true
 deps =
-    .[test,docs]
+    flake8
+    check-manifest
+    check-python-versions
+commands =
+    flake8 src setup.py
+    check-manifest
+    check-python-versions
+
+[flake8]
+doctests = 1
```

Comment:
This needs to be `[tool:flake8]` inside `tox.ini`, AFAIR.  (It's just `[flake8]` in a `setup.cfg` or a `.flake8`, but tox.ini and pyproject.toml namespace their sections)

Reply:
No, I misremembered: flake8 doesn't need the tool: prefix, and it support pyproject.toml at all.

So how come this doesn't work?  Are there multiple `[flake8]` config sections in different files?  IIRC flake8 only looks in the first file where it finds its own config.

... no, I don't see anything like that either.

What is going on here?  If doctests work, why 
`from zope.dublincore.dcsv import encode, decode  # noqa: F401` in that test file?  If doctests don't work, why don't they?

Reply:
Okay, I tried to remove the `# noqa: F401`, and `tox -e lint` works fine.  So this config is okay.